### PR TITLE
Week 14: 1st Attempt

### PR DESCRIPTION
**Pull request by pvo4**
Got it to print the logo, but the file is inside the originals

### CODE REVIEW

**Comment by halfaro1** [comment]
@pvo4 isn't there a `for` loop in the tutorial in the book? I think you're missing that.

**Comment by pvo4** [comment]
that's weird I have the for loop in my file, but it doesn't show it in the commit

File: resizeAndAddLogo.py [hunk 1]
```diff
@@ -7,8 +7,35 @@
 logoIm = Image.open( LOGO_FILENAME )
 logoWidth, logoHeight = logoIm.size
 
+os.makedirs('withLogo', exist_ok=True)
 #TODO: Loop over all files in the working directory
-#TODO: Check if file image needs to be
-#TODO: Calculate the new width and height to resize to.
-#TODO: Add the logo.
-#TODO: Save changes.
+for filename in os.listdir('.'):
```
Comment (halfaro1) [inline resizeAndAddLogo.py]: @pvo4 this for-loop looks through the current working directory. The original files are not in the current working directory, but in the `originals` folder, right?

**Comment by pvo4** [comment]
I added the originals folder to os.lisdir() before, but got nothing

**Comment by halfaro1** [comment]
@pvo4 
The automated testing for this particular assignment isn't working as expected, so I've graded your project manually. Everything is 👍 on your project, so I am closing this and changing the tag to `graded`. When I transfer these grades to Blackboard, you'll see the tag switch to `grade updated on blackboard`. Great job!

**Pull request closed without merging**